sdl-deltalake/pom.xml

```diff
@@ -67,6 +67,10 @@
 			<groupId>org.apache.spark</groupId>
 			<artifactId>spark-catalyst_${scala.minor.version}</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Scope=compile should be enough. And please add a comment maybe even referencing the stackoverflow issue.

Reply:
Thanks, done in https://github.com/smart-data-lake/smart-data-lake/pull/360/commits/70668e7caf04a8106101e92638c749d3e87c410c.

Will wait with the merge until jms fix is finalized.